source/renderer/app/utils/votingPDFGenerator.js

```diff
@@ -62,7 +62,9 @@ export const votingPDFGenerator = async ({
   intl,
 }: Params) => {
   // Consolidate data
-  const title = intl.formatMessage(messages.title, { fundNumber });
+  const title = intl.formatMessage(messages.title, {
+    nextVotingFundNumber: fundNumber,
```

Comment:
Isn't better to rename `fundNumber` into `nextVotingFundNumber` to avoid confusion?